components/textfield/index.css

```diff
@@ -1265,17 +553,10 @@ governing permissions and limitations under the License.
 
 /*** Text Area ***/
 .spectrum-Textfield--multiline {
```

Comment:
It looks like Text Area is its own component now (at least it's a separate page in Figma). Do you think that will change anything in this Text Field component? We do technically return a `textarea` with the multiline variant, so does that encroach at all? I have no idea, I'm just thinking out loud.

Reply:
Taking a quick glance, textarea looks pretty similar (and was done by the same designer), and we have a card in the backlog for [textarea](https://jira.corp.adobe.com/browse/CSS-768), the main differences are the minimum width and height/minimum height.

It also looks like they were designed separately for S1 but are still both part of the textfield component, so I suspect when we get to migrating textarea it would just consist of making any necessary adjustments within the same textfield component.